src/birthder.js

```diff
@@ -33,7 +33,7 @@
     const BIRTHDAY_ANNOUNCEMENT_BEFORE_MODE = process.env.BIRTHDAY_ANNOUNCEMENT_BEFORE_MODE || false;
 
     const MSG_PERMISSION_DENIED = "Permission denied.";
-    const DATE_FORMAT = "DD/MM/YYYY";
+    const DATE_FORMATS = ["DD.MM.YYYY", "D.M.YYYY", "DD/MM/YYYY", "D/M/YYYY", ];
```

Comment:
Yes, single character formats (`"D.M.YYYY", "D/M/YYYY"`) fit such cases:
![bdays2](https://user-images.githubusercontent.com/16405513/44098741-a59b1810-9fe9-11e8-835e-a6a8e4f64681.png)
--------------------------------------------
![bdays3](https://user-images.githubusercontent.com/16405513/44098740-a540dd5a-9fe9-11e8-9853-851a79382503.png)

Reply:
Ok. I wouldn't expand this list with `"DD.MM.YYYY"` and `"D.M.YYYY"`, because the help claims the script uses only the `<date>/<month>/<year>` format.

Reply:
We can update help text in next PR =)